### ISSUE TITLE
Unable to start server with 12.3 postgres, fresh installation

### ISSUE THREAD

**Issue by rwchunk** [happypandax/happypandax#175]
**Describe the issue**
Server fails to start.

**To Reproduce**
1. Download and unpack 12.3
2. Configure 12.3 to use postgres with a new database
3. Start server, encounter error

**Expected behavior**
Server starts without error.

**Logs**
[error.log](https://github.com/happypandax/happypandax/files/3786400/error.log)
[debug.log](https://github.com/happypandax/happypandax/files/3786401/debug.log)

**Environment:**
 - OS: Windows 10 LTSC
 - HPX Version 12.3

**Comment by twiddli** [comment]
from the logs, the database doesn't seem to be new

**Comment by rwchunk** [comment]
[activity.log](https://github.com/happypandax/happypandax/files/3792889/activity.log)
[error.log](https://github.com/happypandax/happypandax/files/3792890/error.log)
I redid it

**Comment by twiddli** [comment]
I just tested it myself and on a new database it shouldn't say:
```
Oct-10 03:54:34--INFO   [database].happypanda.core.db: Using postgres database 
Oct-10 03:54:35--INFO   alembic.runtime.migration: Context impl PostgresqlImpl.
Oct-10 03:54:35--INFO   alembic.runtime.migration: Will assume transactional DDL.
Oct-10 03:54:35--INFO   [database].happypanda.core.db: Database update found. Updating... 
```
like it's saying in your log. If it really does that then that's a bug.
Are you creating the database yourself prior to running HPX?

**Comment by rwchunk** [comment]
I created new test databases through pgadmin each time, it should be empty right?
That line still appears in the build you provided earlier. Testing with a new DB ofc.
logs:
[activity.log](https://github.com/happypandax/happypandax/files/3799811/activity.log)
[error.log](https://github.com/happypandax/happypandax/files/3799812/error.log)

**Comment by twiddli** [comment]
in the log it says it's HPX `0.10.1`

**Comment by rwchunk** [comment]
Using the actual 12.4, there were no errors, and that line does not appear in the logs.

**Comment by twiddli** [comment]
fixed in `v0.12.4`
